### PR TITLE
Adopt nJy in blackbody model and eclipsing binaries

### PR DESCRIPTION
**Pull request by hombit**
This fixes periodic model to return values in nJy, and adds some comments to black-body model to highlight that it has CGS inputs and outputs.

### CODE REVIEW

File: src/tdastro/consts.py [hunk 1]
```diff
@@ -20,3 +20,10 @@
    where
    g(x,y) = 1 / (2π sigma²) exp(-(x²+y²)/2/sigma²).
 """
+
+ANGSTROM_TO_CM = (1.0 * u.AA).to_value(u.cm)
+"""Conversion factor from Angstrom to cm, 1e-8"""
+
+
+CGS_FNU_UNIT_TO_NJY = (1.0 * u.erg / u.second / u.cm**2 / u.Hz).to_value(u.nJy)
+"""Conversion factor from erg/s/cm²/Hz to nJy, 1e23"""
```
Comment (mi-dai) [inline src/tdastro/consts.py:29]: Do these consts need tests?

**Review by mi-dai — COMMENTED** [review]
Looks good! Only one minor comment

File: src/tdastro/consts.py (hunk 1 above)
Comment (hombit) [reply src/tdastro/consts.py:29]: Done

Comment (mi-dai) [inline src/tdastro/consts.py:29]: 1e32

**Review by mi-dai — CHANGES_REQUESTED** [review]
One more change in docstring!

File: src/tdastro/consts.py (hunk 1 above)
Comment (hombit) [reply src/tdastro/consts.py]: Fixed!

**Review by mi-dai — APPROVED** [review]

**Pull request merged**